database/sql/V059__add_block_number_order_execution.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE order_execution
+    ADD COLUMN block_number bigint NOT NULL DEFAULT 0;
```

Comment:
The column should probably drop the `DEFAULT` modifier after the migration populated the old entries.

Reply:
dropped `DEFAULT` but added `COALESCE(settlements.block_number, 0)` so that we have `0` inserted if there is no corresponding auction_id in `settlements`.